OneSignalSDK/onesignal/build.gradle

```diff
@@ -56,6 +56,10 @@ dependencies {
     // firebase-messaging:17.6.0 is the max version since we still have code looking for FirebaseInstanceIdService
     api 'com.google.firebase:firebase-messaging:[10.2.1, 17.3.99]'
 
+    // Huawei PushKit
+    // KEEP as "compileOnly", so OneSignal isn't a direct dependency in the POM file.
+    compileOnly 'com.huawei.hms:push:4.0.2.300'
```

Comment:
Should we include a range here or keep it specific for now?

Reply:
Since this is `compileOnly` let's not use a version range here so builds are consistent.